src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -166,7 +168,14 @@ public static TypedData ToRpc(this object value, ILogger logger)
                             var bytes = new byte[length];
                             request.Body.Read(bytes, 0, length);
                             body = bytes;
-                            rawBody = Encoding.UTF8.GetString(bytes);
+                            if (Capabilities.IsCapabilityEnabled(ExposedCapabilities.RawHttpBodyBytes))
```

Comment:
Do not user static class. Pass the capabilities of the languageworkerchannel

Reply:
Will address when opening a separate PR for adding raw body bytes capability.